.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
```

Comment:
I'm not sure we need to force people to use some specific styling, maybe we should remove this one?

Reply:
@dotansimha 
This is the default styling when user create worker from CF. But I can remove it and make it more clean. WDYT?